model/src/commonMain/kotlin/io/github/droidkaigi/confsched2020/model/FloorMap.kt

```diff
@@ -1,6 +1,17 @@
 package io.github.droidkaigi.confsched2020.model
 
-enum class FloorMap(val title: String) {
-    FLOOR_1("1F"),
-    FLOOR_5("5F")
+enum class FloorMap(val id: Int) {
```

Comment:
According to [`Room.kt`](https://github.com/DroidKaigi/conference-app-2020/blob/master/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2020/model/Room.kt#L11), we have Exhibition room which has `0` for the room id.

I think this enum <-> value mapping is very useful for other features so this definition can be located in `Room.kt`, and then we can pass the enum directly via navArgs (navArgs supports enums. ref.https://developer.android.com/guide/navigation/navigation-pass-data#supported_argument_types).

Reply:
@KeithYokoma 
In other words, it seems better to move this roomId information to the Room class. Is it like that?? 👀

Reply:
Good catch. Yeah it's nice to have Room IDs as constants in `Room` class.

Reply:
@KeithYokoma @takahirom 
Thank you for your review ! 
I fixed with fed9907bb3e244b4f830db9efd7654b8189ef6c9
FloorMap changed to RoomType